src/audio/Audio.js

```diff
@@ -78,13 +78,20 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
+		if ( when === undefined || when === 0 ) when = this.context.currentTime;
+		this.startTime = offset !== undefined ? offset : this.startTime;
```

Comment:
I'm not sure either but think overwriting `startTime` makes the state more consistent vs retaining some old value.

e.g. without it, this would happen:
```
audio.play();
// ...
audio.pause();
audio.play( 0, 4.2 );
// ...
audio.play() // <-- starts from previous pause position
```

On a side note, it's a bit weird that `startTime` doesn't reset in `onEnded` (but it does in `stop`):
```
audio.play();
// ...
audio.pause();
audio.play();
// ... after the audio ended
audio.play(); <-- starts from the last pause position
```

But I guess this is because `startTime` is used as both a internal property for tracking the pause position as well as a public API to set the start time.